polynote-runtime/src/main/scala/polynote/runtime/ValueRepr.scala

```diff
@@ -0,0 +1,249 @@
+package polynote.runtime
+
+import java.nio.ByteBuffer
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.atomic.AtomicInteger
+
+import sun.misc.Cleaner
+
+/**
+  * A plain text representation of a value
+  */
+final case class StringRepr(string: String) extends ValueRepr
+
+/**
+  * A MIME representation of a value. Binary values must be base64-encoded.
+  */
+final case class MIMERepr(mimeType: String, content: String) extends ValueRepr
+
+/**
+  * A binary representation of a value, encoded in Polynote's format (TODO)
```

Comment:
Is this still TODO? Looks like ReprsOf does handle encoding?

Reply:
You're right, it's not exactly TODO anymore